### PR TITLE
Project 5 on Microverse: Design Teardrop

### PR DESCRIPTION
**Pull request by 300ms**
We have printed out a simple blueprint of webpage given to us. And built up the heatmap of it by using gray-scale colors. This is a review of our personal vision and used as much as less elements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            <div class="article-7">
+                <article class="article-2-1 grid-article-5-1"><h2 class="text-article-2">It has roots in a piece of classical Latin literature</h2></article>
+                <article class="article-5-1 grid-article-5-2"><h2 class="text-article-3-1">Also the leap into electronic</h2></article>
+            </div>
+            <div class="see-articles">
+                <div>Why do we use it?</div>
+            </div>
+        </main>
+        <footer>
+            <h1>The standard Lorem Ipsum passage</h1>
```
Comment (samgaco) [inline index.html:110]: The footer should contain centered links

**Comment by samgaco** [comment]
Reviewing

File: index.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            <nav class="nav-bar">
+                <ul class="nav-ul">
+                    <li class="nav-ul-elements">Neque</li>
+                    <li class="nav-ul-elements">porro </li>
+                    <li class="nav-ul-elements">quisquam </li>
+                    <li class="nav-ul-elements">est qui </li>
+                    <li class="nav-ul-elements">dolorem </li>
+                </ul>
+            </nav>
+
```
Comment (samgaco) [inline index.html]: The nav bar should contain `<div>` containers

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi  @300ms and @Luzaks :high_brightness: 

#### Things to address before the approval:

1. The footer should contain centered links. 
2. Address errors given to you by the[ css validator.](https://jigsaw.w3.org)
3. The nav bar should contain `<div>` tags

------------------

* **[OPTIONAL]** Add some text to the body of the articles (there is only titles so far)

------------------


### Have into account :arrow_down: 

* Friendly reminder: You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in this project. However, changing it now would take too much time. Please make sure you use [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project. 

* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

--------------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Review by Ispirett — APPROVED** [review]
Hey, @Luzaks @300ms Great Job with Changes.
image. 

**You can merge this project now!!!** 

**[Optional] for the following**
- [ ] You can add a block that represents an image  for the section under the section with the following text 
`Contrary to popular belief, Lorem Ipsum is not simply random text. It has`
- [ ] The footer you can give the  `a` links color that represents the contrast of that section like white

Github flow reminder.
Friendly reminder: You should use Github Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

**Pull request merged**